news/2020-04-10-osu-spotlights-new-team-and-league-system.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "osu! Spotlights: New Team and League System"
+date: 2020-04-13 08:00:00 +0000
+---
+
+With our latest Spotlight season released very recently, we are underway reworking osu! Spotlights from ground up, starting with a brand new team and adding a whole new league system.
+
+![](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+Since March 2017, osu! Spotlights have worked in their current shape and form. While many mappers and players already love the idea of beatmaps being put on a pedestal, we want to expand on the idea by giving the curation and participation in osu! Spotlights a whole new twist.
+While we want to keep recommending beatmaps to you, we also want to add a whole new competitive and seasonal league closely attached to the recommended beatmaps. With this, you and your friends will have even more reason to delve deep into the best creations osu! has to offer.
+
+We are still working on the details about how this league will work, but here are some aspects we *plan* to do:
+
+- **We want to host weekly timeshift lobbies**, using osu!lazer's recently implemented feature, by having people compete actively on a portion of the beatmaps we have selected for the Spotlights.
+- With hosting weekly multiplayer lobbies, **we will release our recommendations little by little**. To keep things fresh, maps being played in the lobby will change every week. Because of this, we will only publish portions of the whole Beatmap Spotlights every week.
```

Comment:
is spotlights still seasonal then? will there be more spotlighted maps overall if new ones come every week? should have answers (even if they're tentative) to these questions in the post